openide.filesystems/src/org/openide/filesystems/JarFileSystem.java

```diff
@@ -503,24 +503,16 @@ private InputStream getTemporaryInputStream(JarFile jf, JarEntry je, boolean for
         if (createContent || forceRecreate) {
             // JDK 1.3 contains bug #4336753
             //is = j.getInputStream (je);
```

Comment:
The commented code and the comment can be removed.

Reply:
Are you sure?
I left it there, because the code underneath still calls the 'getInputStream4336753()'.
I think this is really obsolete and the entire method can be deleted, but I wanted to make this PR focused on one thing.

Reply:
Agreed - I misread the change. Sorry for the noise.